pgdb/v1/schema.go

```diff
@@ -39,6 +42,13 @@ func CreateSchema(msg DBReflectMessage) ([]string, error) {
 	}
 
 	_, _ = buf.WriteString(")\n")
+
+	if desc.IsPartitioned() {
+		_, _ = buf.WriteString("PARTITION BY LIST(")
```

Comment:
I think goqu doesn't have great support for these kinds of statements, but i really wonder if we should at least be doing prepared statements instead of inline-concat'ing the tenant id?

Reply:
Yeh I think prepared statements would be better so I'm gonna work converting this over and see how it goes

Reply:
Seems like `PREPARE` isnt _recommended_ for create statements but its not impossible. I think leaving it as is for now is fine. It works. If we want to look into constructing `PREPARE` statements in the future it MIGHT be helpful but for right now I don't think its really needed. Since we're just creating the schema I dont think it needs to be optimized since we generally dont do this often like we do selects / inserts

Reply:
sorry i didn't mean "prepare" prepared.

I meant paramaterized queries, rather than sprintf / string concat.

Reply:
We still need sprintf for the table names because otherwise it would be in quotes but I could at least parameterize the tenantId https://github.com/ductone/protoc-gen-pgdb/pull/63/commits/7fca34a8d4c53f4053ff19142d0caf1c686092f7